src/views/createLinks.js

```diff
@@ -0,0 +1,12 @@
+import { quizData } from '../data.js';
+import createDOMElement from '../utils/createDOMElement.js'
+const createLinks = () => {
```

Comment:
It looks like this view isn't actually used?

I think you should use it, it's nicer to have it in a function like this than to put it straight into `createQuestionElement` like you've done now.